src/components/Firestore/Field/DocumentStore.tsx

```diff
@@ -0,0 +1,106 @@
+/**
```

Comment:
nit: consider renaming to `store.tsx`? We don't actually have a `DocumentStore` class.

Reply:
Done